src/types/input-configs.ts

```diff
@@ -55,9 +55,15 @@ export interface InputTextareaConfig extends InputBaseConfig, TextareaConfig, Re
     type: "textarea";
 }
 
+export interface CheckboxConfig extends InputBaseConfig {
```

Comment:
`CheckboxConfig` must not extends from `InputBaseConfig`. You can see `TextareaConfig` as an example.